pom.xml

```diff
@@ -90,6 +92,7 @@
         <version.jmh>1.21</version.jmh>
         <version.mockito>3.0.0</version.mockito>
         <version.awaitility>3.1.6</version.awaitility>
+        <version.testcontainers>1.12.3</version.testcontainers>
```

Comment:
```suggestion
        <version.testcontainers>1.12.5</version.testcontainers>
```

Reply:
I couldn't apply it, as it's not "my" PR, but I did this change. Thanks!